examples/hosts/app-integration/external-data/src/model/appModel.ts

```diff
@@ -28,4 +29,11 @@ export class AppModel extends TypedEventEmitter<IAppModelEvents> implements IApp
 	public readonly sendCustomDebugSignal = (): void => {
 		this.runtime.submitSignal("debugSignal", { type: "ExternalDataChange" });
 	};
+
+	/**
+	 * {@inheritDoc IAppModel.containerResolvedUrl}
+	 */
+	public readonly containerResolvedUrl = (): IFluidResolvedUrl => {
```

Comment:
We need to cast it as IFluidResolvedUrl because then it complains that IWebResolved doesn't have url, but the object definitely does have url and is a IFluidResolverUrl. Not sure why it thinks it's not.